test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp

```diff
@@ -214,3 +213,70 @@ TEST_VM(CommittedVirtualMemoryTracker, test_committed_virtualmemory_region) {
   }
 
 }
+
+#ifndef _WINDOWS
```

Comment:
Test should work on Windows, too

Reply:
On Windows `os::committed_in_range` uses `VirtualQuery` which returns only whether the region is `MEM_COMMIT`. [A region that is set to `MEM_COMMIT` is may not yet be faulted in.](https://learn.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-virtualalloc) So unlike mincore, it's not actually useful for getting a fine-grained measurement of physical memory used, although it's helpful to find the parts of the thread stacks that are committed. 

I did a quick check on my windows computer by committing a big amount of memory and checking the RSS. `os::committed_in_range` reports the memory as committed, but it's not reflected in the RSS. When I actually touch the chunk of committed memory, `os::committed_in_range` reports the same as before, but I can see it in the RSS this time.